datalad/support/gitrepo.py

```diff
@@ -98,6 +104,7 @@
 
 
 lgr = logging.getLogger('datalad.gitrepo')
+cfg = ConfigManager()
```

Comment:
This is not needed. Just use `self.config` below.

Reply:
thanks, done!